### ISSUE TITLE
Branch list filters

### ISSUE THREAD

**Issue by acdha** [laullon/gitx#160]
It would be extremely nice if GitX could be configured not to show merged branches by default - since I use per-ticket branches for many tasks I have a large branch list where most of the entries are moot.

**Comment by dgm** [comment]
Yes, and the ability to filter which branches to show in the main flow area...
